content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/magnetic_dipole_definition/index.rst

```diff
@@ -0,0 +1,192 @@
+.. _definition_magnetic_dipole_index:
+
+Defining the Magnetic Dipole
+============================
+
+.. Purpose::
+
+    Here, we provide a physical description of the magnetic dipole.
+    This is used to develop a mathematical expression which can be used to replace the magnetic source term in Maxwell's equations.
+
+
+**General Description**
+
+
+.. figure:: images/H_source_magnetic_dipole.png
+		:align: right
+		:figwidth: 50%
+		:name: MagDipole_index
+		
+		Physical representation of the magnetic dipole source where :math:`\mathbf{m}` = 1 Am :math:`\!^2`.
+
+
+
+
+There are two commonly used models for the magnetic dipole.
+The first model describes the magnetic dipole as an infinitessimally small volume of magnetized material (i.e. a very small bar magnet).
+The second model describes the magnetic dipole using an infinitessimally small current loop.
+In both cases, the strength of the magnetic dipole source is defined by a dipole moment (:math:`\mathbf{m}`).
+This leads to a magnetic source term (:math:`\mathbf{J_m^s}`) of the form:
+
+
+.. math::
+	\mathbf{J_m^s} = - i\omega \mu \mathbf{m} \delta (x) \delta (y) \delta (z)
+	:name: Jm_def
+
+
+where :math:`\delta (x)` is the Dirac delta function.
+The magnetic dipole source is responsible for generating a primary magnetic field in the surrounding region; secondary electric and magnetic fields are discussed later.
+This is illustrated in :numref:`MagDipole_index`.
+
+
+
+**Magnetized Volume Model**
+
+
+This model derives the magnetic dipole source by considering a volume of uniformly magnetized material; in other words, a bar magnet.
+Let us assume the volume has uniform magnetization (:math:`\mathbf{M}`) and has dimensions :math:`\Delta x`, :math:`\Delta y` and :math:`\Delta z`; giving it a volume of :math:`\Delta V`.
+The resulting magnetic source term (:math:`\mathbf{J_m^s}`) is given by:
+
+
+.. math::
+	\mathbf{J_m^s} = - i\omega \mu \mathbf{M (r)}
+	:name: Jm_M
+	
+
+where
+
+.. math::
+	\begin{split}
+	\mathbf{M (r)}\! =\! \mathbf{M} \Delta V & \!\Bigg [ \! \frac{u \big ( x \! +\!\frac{\Delta x}{2} \big ) \! - \! u \big ( x \! -\!\frac{\Delta x}{2} \big )}{\Delta x} \! \Bigg ] ... \\ 
+	& \;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\; \! \Bigg [ \! \frac{u \big ( y \! +\!\frac{\Delta y}{2} \big ) \! - \! u \big ( y \! -\!\frac{\Delta y}{2} \big )}{\Delta y} \! \Bigg ] \! 
```

Comment:
This is some pretty wild formatting, which would make it very hard for someone to edit or reuse. Can the `\;` be simplified by using something like `\quad`? and perhaps moving the alignment character later on in the first line?